image/base/Dockerfile

```diff
@@ -145,14 +143,6 @@ RUN gem install pups --force &&\
 ADD thpoff.c /src/thpoff.c
 RUN gcc -o /usr/local/sbin/thpoff /src/thpoff.c && rm /src/thpoff.c
 
-# clean up for docker squash
```

Comment:
We no longer use docker squash - that has been gone for a long time. Here, we now desire /tmp to stick around as a way of keeping /tmp/install-redis around for installing later within a build for discourse-dev.

Reply:
I wonder if we could delete these after the `RUN` command that generates them to keep the image small.

Reply:
The other option we have is to add a tmpfs have a for these during the build.

Note that I'm keeping /tmp around to ensure install-redis remains as needed - it's called in both this image as well as the dev image.